### DISCUSSION TITLE
build

### DISCUSSION THREAD

**Discussion by Pianaixiaoyu** [opentiny/tiny-engine#400] [General]
A package deployment appears        Uncaught SyntaxError: Invalid or unexpected token (at standalone.mjs:1:4723)

**Comment by wenmine** [comment]
提供的信息不足，请提供具体的复现场景、命令，以及相关配置@Pianaixiaoyu

**Comment by chilingling** [comment]
该问题是由于 CDN 源 https://npm.onmicrosoft.cn  代理的 prettier 源码出现了问题，切换成其他 CDN 源如 https://unpkg.com 即可解决

@Pianaixiaoyu
